src/graphql-server/schema.graphql

```diff
@@ -34,6 +34,7 @@ type HostPricing {
 
 type HolofuelUser {
   id: ID
+  pubkey: String
```

Comment:
why do we need `pubkey`? I'm not seeing where we're using it.

Reply:
It's used for the HolofuelCounterparty Query, instead of an ID as a current solution preventing caching concerns.